src/Apps/Order/redirects.tsx

```diff
@@ -183,6 +183,10 @@ export const redirects: RedirectRecord<OrderQuery> = {
         goToShippingIfShippingIsNotCompleted,
       ],
     },
+    {
+      path: "payment/new",
+      rules: [goToShippingIfShippingIsNotCompleted],
```

Comment:
Since you can only access the `payment/new` flow if you've already submitted an offer/order, I _think_ we'll want to change this to:

```
rules: [goToStatusIfNotOfferOrder, goToStatusIfOrderIsNotSubmitted]
```

Reply:
done